test/e2e/config/e2e_conf.yaml

```diff
@@ -205,12 +205,21 @@ variables:
   IMAGE_CHECKSUM_TYPE: "sha256"
   IMAGE_USERNAME: "metal3"
   NODE_DRAIN_TIMEOUT: "0s"
+  IRONIC_RELEASE_23.1: "data/ironic-deployment/overlays/release-23.1"
+  IRONIC_RELEASE_24.0: "data/ironic-deployment/overlays/release-24.0"
+  IRONIC_RELEASE_24.1: "data/ironic-deployment/overlays/release-24.1"
+  IRONIC_RELEASE_LATEST: "data/ironic-deployment/overlays/release-latest"
+  BMO_RELEASE_0.3: "data/bmo-deployment/overlays/release-0.3"
```

Comment:
Surely we don't have BMO 0.3 support? 0.4 is the last CI tested version, 0.5 is the last supported one?

Reply:
yes, https://book.metal3.io/version_support#baremetal-operator